eng/pipelines/templates/variables/globals.yml

```diff
@@ -2,4 +2,7 @@ variables:
   DotNetCoreSDKVersion: '3.0.100-preview5-011568'
   DotNetCoreRuntimeVersion: '2.1.10'
   OfficialBuildId: $(Build.BuildNumber)
-  skipComponentGovernanceDetection: true
\ No newline at end of file
+  skipComponentGovernanceDetection: true
+  BuildConfiguration: ''
+  ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    BuildConfiguration: 'Release'
```

Comment:
Have you tested that this second assignment actually works in an internal build?